Dockerfile

```diff
@@ -12,6 +36,7 @@ RUN fc-cache -fsv
 
 # Install PhantomJS (used by cyhy-bod-scorecard and potentially by cyhy-cybex-scorecard); may not be needed in the future
 RUN apt-get update && apt-get -y install \
+    curl \
     build-essential chrpath libssl-dev libxft-dev \
     libfreetype6 libfreetype6-dev \
     libfontconfig1 libfontconfig1-dev
```

Comment:
I think these should all be on their own lines to keep the formatting style the same as similar tasks.

Reply:
Yessir, good call- fixed in https://github.com/cisagov/cyhy-reports/pull/57/commits/3b2b776c7170b164c259b7a25dcfe166e6a6388a.